examples/kubernetes/pod_iam_impersonation/README.md

```diff
@@ -0,0 +1,83 @@
+## CSI Driver POD IMPERSONATION
+
+This example shows how you can use the pod impersonation feature available 
+in the CSI Driver to enforce access control when mounting EFS Access point protected with EFS resource policy.
+This feature leverages IAM roles for service accounts (IRSA) and your EKS cluster must be enabled for IRSA. 
```

Comment:
it should be stated explicitly that this feature is exclusive to EKS. right now it's implied because you need IRSA and only EKS has IRSA

Reply:
Fixed, specified that it only work with EKS.